### ISSUE TITLE
Documentation unclear for namecheap support

### ISSUE THREAD

**Issue by clarjon1** [troglobit/inadyn#361]
Hello,

I am looking to start using inadyn for dynamic dns updates with namecheap

However, there appears to be conflicting information in the documentation. The documentation simultaneously states that namecheap is fully supported, but the only example shown is via the custom section, which is meant for use for unsupported services.

Could the documentation be updated with an example of namecheap using the supported solution instead of what appears, per the documentation, to be an out of date configuration method?

Thanks!

**Comment by troglobit** [comment]
The project is open to help and pull request to update the documentation.

**Comment by clarjon1** [comment]
Well, as a user who has never used this software before, I'm afraid I can't be much help when I'm relying on documentation to learn how to use it properly. Hopefully someone who has used this with namecheap recently will happen to check the github users to fix this, as there's a good chance that if I tried submitting with my current level of knowledge, i may introduce insecure or inaccurate information.

I did try looking in the code but I could not find any code specific references to namecheap, at least thru github's search. Due to my unfamiliarity with the project, I may very well be trying to find this info the wrong way, but I've no idea if I am or not.
